pkg/permissions/config.go

```diff
@@ -22,4 +27,7 @@ func MustViperFlags(v *viper.Viper, flags *pflag.FlagSet) {
 
 	flags.Bool("permissions-ignore-no-responders", false, "ignores no responder errors when auth relationship requests are published")
 	viperx.MustBindFlag(v, "permissions.ignoreNoResponders", flags.Lookup("permissions-ignore-no-responders"))
+
+	flags.Bool("default-allow", false, "grant permission checks when url is not set")
```

Comment:
Should this be `permissions-default-allow`?

Reply:
oh good catch, thank you.